profiles/base/package.use

```diff
@@ -31,3 +31,5 @@ app-admin/sudo -ldap
 # Bug #295615 - Disable cxx by default, in order to avoid a die
 # in pkg_setup with default USE settings (cxx requires sasl).
 net-nds/openldap -cxx
+
+sys-apps/iproute2 elf
```

Comment:
It is ok-ish, but,
if we add Flatcar-specific changes in portage-stable, then it could be overwritten by upstream changes, next time when we sync with Gentoo for example.
That's why we usually gather all the Flatcar-specific profile changes in `coreos-overlay/profiles/`.

Reply:
I'd rather not put USE flags for packages in `portage-stable` into the profile of `coreos-overlay`. Profiles exist per overlay and should cover packages in the respective overlay.
You're rightfully pointing out that we need to pay attention when merging Gentoo changes though. We'll need to tread carefully there.

Reply:
I am not sure I completely agree, but anyway let's merge it for now.